packages/shared/sierra-client/tests/http-sierra-client.test.ts

```diff
@@ -64,7 +65,8 @@ describe('HTTP sierra client', () => {
       const response = await client.createPatron(
         'Ravioli',
         'Ravi',
-        'raviravioli@ppastatimestest.com'
+        'raviravioli@ppastatimestest.com',
```

Comment:
looks like ravi has typo-ed their email?

Reply:
That's a bad pasta move eh?